include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -320,21 +308,11 @@ class sequence_file_format_embl
                 bp = std::min(sequence_size, bp + 60);
                 ++i;
                 stream_it = ' ';
-                if (bp % 60 > 0)
-                {
-                    for(int j = bp; j < (60 * i); j++)
-                    {
-                        stream_it = ' ';
-                        if (j % 10 == 0)
-                            stream_it = ' ';
-                    }
-                    ranges::copy(std::to_string(bp), stream_it);
-
-                }
-                else
-                {
-                    ranges::copy(std::to_string(bp), stream_it);
-                }
+                uint8_t num_blanks = 60 * i - bp;  // for sequence characters
+                num_blanks += num_blanks / 10;     // additional chunk separators
+                for(int j = 0; j < num_blanks; j++)
+                    stream_it = ' ';
```

Comment:
Please avoid unnecessary loops. See the interface of the std::string constructor (2) [here](https://en.cppreference.com/w/cpp/string/basic_string/basic_string) and use for instance:
```cpp
ranges::copy(std::string(num_blanks, ' '), stream_it);
```
Probably there is also a solution with the ranges library, if you prefer.